src/OrbitGl/ViewportTest.cpp

```diff
@@ -151,20 +164,26 @@ void VerifyConversion(Viewport& viewport, const Vec2i& screen_pos, const Vec2& w
   EXPECT_EQ(viewport.WorldToScreenWidth(world_size[0]), screen_pos[0]);
 
   EXPECT_EQ(viewport.QtToGlScreenPos(screen_pos),
-            Vec2i(screen_pos[0], viewport.GetHeight() - screen_pos[1]));
+            Vec2i(screen_pos[0], viewport.GetScreenHeight() - screen_pos[1]));
 }
 
 TEST(Viewport, CoordinateConversion) {
   Viewport viewport(10, 100);
-  // Extents are large enough to scroll
+  // Extents are large enough to scroll.
   viewport.SetWorldExtents(500, 500);
-  viewport.SetWorldWidth(20.f);
-  viewport.SetWorldHeight(50.f);
 
-  // NOTE: World space is y-down, but (0,0) is top left
   Vec2i screen_pos = Vec2i(8, 20);
-  Vec2 world_pos = Vec2(16.f, -10.f);
-  Vec2 world_size = Vec2(16.f, 10.f);
+  Vec2 world_pos = Vec2(8.f, -20.f);
+  Vec2 world_size = Vec2(8.f, 20.f);
+  VerifyConversion(viewport, screen_pos, world_pos, world_size);
+
+  // Change zoom: Zoom out 100% horizontally, zoom in 50% vertically.
```

Comment:
200% horizontally

Reply:
*By* 100% was what I wanted to say here. But this is not clear, I'll change the comment, thanks!

Reply:
I've just seen "zoom-out". Still prefers 200% and 50%.

Reply:
Changed